src/Var.jl

```diff
@@ -302,6 +312,20 @@ function is_z_1D(var::OutputVar)
     return length(size(altitudes(var))) == 1
 end
 
+"""
+    isempty(var::OutputVar)
+
+Determine whether an OutputVar is empty.
+"""
+function Base.isempty(var::OutputVar)
+    # Do not include :interpolant because var.interpolant is Nothing if data is
+    # zero dimensional and empty and isempty(Nothing) throws an error
+    return map(
+        fieldname -> isempty(getproperty(var, fieldname)),
+        [:attributes, :dims, :dim_attributes, :data, :dim2index, :index2dim],
+    ) |> all
```

Comment:
```suggestion
    return map(
        fieldname -> isempty(getproperty(var, fieldname)),
        filter(x -> x != :interpolant), fieldnames(OutputVar))
    ) |> all
```
So that this will keep working if we add more fields